packages/eas-cli/src/build/ios/credentials.ts

```diff
@@ -7,6 +7,8 @@ import IosCredentialsProvider, {
   IosCredentials,
 } from '../../credentials/ios/IosCredentialsProvider';
 import { getAppLookupParamsFromContext } from '../../credentials/ios/actions/BuildCredentialsUtils';
+import { IOSCapabilitiesOptions } from '../../credentials/ios/appstore/ensureAppExists';
```

Comment:
```suggestion
import { IosCapabilitiesOptions } from '../../credentials/ios/appstore/ensureAppExists';
```
With the exception of `Platform.IOS` we use `Ios` everywhere in cli